samples/tfm_integration/CMakeLists.inc

```diff
@@ -27,7 +27,7 @@ else()
 endif()
 
 # Select a known trusted-firmware-m commit
-set(TFM_COMMIT_HASH "5358c8c4cf385cde6091adc74dce53fa3f0c2ce4")
+set(TFM_COMMIT_HASH "4117e0353c55dce739f550515c5b007411142ba7")
```

Comment:
This could specific to a tag I think. TF-Mv1.0-RC2
I may update this in previous patch.